README.md

```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
    - A robust foundation model, trained on 117 classes encompassing various organs and tumors.
    - The model's training and validation involved CT images from over 4,000 patients, ensuring its high accuracy and reliability for medical image analysis.
 
-### How to Get Started
+4. Dicon-web Setup and Dataset Configuration:
+   - [Dicom-web Overview](#dicomweb-overview)
+   - [Notebook: Dataset Creation and Experiment Selection](https://github.com/NVIDIA/monai-cloud-api/blob/main/notebooks/Dataset%20Creation%20and%20Experiment%20Selection.ipynb)
+
+5. OHIF Plugin and Overview:
+   - [Getting Started with OHIF Plugin](#getting-started-with-ohif-plugin)
+   - [Setup Customized OHIF ](https://github.com/NVIDIA/monai-cloud-api/tree/main/plugins/ohif)
+   - [OHIF Applications and UI Demo](#basic-ohif-instruction)
+   - [Interactive Annotation](#interactive-annotation)
+   - [Notebook: Annotation and OHIF Overview](https://github.com/NVIDIA/monai-cloud-api/blob/main/notebooks/Annotation%20and%20Continuous%20Learning%20Overview.ipynb)
+
+
+## How to Get Started
 
 To begin using the NVIDIA MONAI Cloud APIs, you can sign up for early access by filling out this survey: [NVIDIA MONAI Cloud API Early Access](https://developer.nvidia.com/nvidia-monai-cloud-api-early-access-program/join). 
+
+## Getting Started with OHIF Plugin
```

Comment:
If we folllow the style @zephyrie 's `integrated_doc` branch, I would suggest to move the contents of OHIF and DICOMweb to two markdown files.

Reply:
Yeah, we should break this down so people can find the relevant section and we should not have this in the README but in the docs section for How-To guides. 
"DICOMWeb Setup" and "Using OHIF and MONAI Cloud API Plugin", we can also move the instructions from the OHIF plugin setup directory to another docs page "Setting up OHIF and MONAI Cloud API Plugin"

Reply:
Also, can we update the plugin name / any other displayed name to "NVIDIA MONAI Cloud API"? We should stop using MONAI Service for any naming moving forward.